praw/models/reddit/redditor.py

```diff
@@ -293,6 +295,22 @@ def moderated(self) -> List["praw.models.Subreddit"]:
                 print(subreddit.display_name)
                 print(subreddit.title)
 
+        .. note::
+
+            A ``prawcore.ServerError`` may be raised if the redditor moderates a large
+            number of subreddits. If that happens, try switching to :ref:`read-only mode
+            <read_only_application>`. For example,
+
+            .. code-block:: python
+
+                reddit.read_only = True
+                for subreddit in reddit.redditor("reddit").moderated():
+                    print(str(subreddit))
+
+            When this method is used in read-only mode, it will not retrieve information
+            about subreddits that require special permissions to access, e.g., private
+            subreddits and premium-only subreddits.
+
```

Comment:
```suggestion

        .. seealso::

             :meth:`.Redditor.moderator_subreddits`
```
Could we add a reference here as well?